README.md

```diff
@@ -94,6 +94,24 @@ If a non-existing video ID is passed into #find, Funky::ContentNotFound will be
 Funky::Video.find('doesnotexist') # => raises Funky::ContentNotFound
 ```
 
+### Use #find_by(url: url) to get a single video from a url
+
+```ruby
+video = Funky::Video.find_by(url: 'https://www.facebook.com/video.php?v=10154439119663508')
+video.id            # => '10154439119663508'
+video.like_count    # => 1169
+video.comment_count # => 65
+video.share_count   # => 348
+video.view_count    # => 10121
+```
+
+If a non-existing video url is passed into #find_by, Funky::ContentNotFound will be raised.
+
+```ruby
+Funky::Video.find_by(url: 'https://www.facebook.com/video.php?v=doesnotexist')
```

Comment:
Just as a thought, in ActiveRecord `User.find_by(id: -234)` returns `nil`, does not raise an error.